README.md

```diff
@@ -31,7 +31,7 @@ Run `make` to build `./bin/firehose.so`. Then use with Fluent Bit:
 * `time_key`: Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis.
 * `time_key_format`: [strftime](http://man7.org/linux/man-pages/man3/strftime.3.html) compliant format string for the timestamp; for example, `%Y-%m-%dT%H:%M:%S%z`. This option is used with `time_key`. You can also use `%L` for milliseconds and `%f` for microseconds. If you are using ECS FireLens, make sure you are running Amazon ECS Container Agent v1.42.0 or later, otherwise the timestamps associated with your container logs will only have second precision.
 * `replace_dots`: Replace dot characters in key names with the value of this option. For example, if you add `replace_dots _` in your config then all occurrences of `.` will be replaced with an underscore. By default, dots will not be replaced.
-
+* `multiple_log_events_per_record`: Option to allow plugin send multiple log events in the same record if current record not exceed the maximumRecordSize(1MiB), default to be `false`, set to `true` to enable this option.
```

Comment:
Let's call this option `simple_aggregation`. And then the description should note that it joins together as many log records as possible into a single Firehose record and delimits them with newline.

Reply:
Also may be add that they should consider increasing the Fluent Bit flush interval in the service section to increase the number of logs sent per flush (to get better aggregation).

Reply:
Wondering, why this can't be enabled by default. Why do we need a settings. Whats the complexity if we do this as defult behavior?

Reply:
Setting it by default would break backwards compatibility with the old behavior...

Reply:
ALSO- this option is only for some customers/use cases. If the log events are all joined together into one record, that only works for some Firehose destinations like S3. For other like elasticsearch, concatenating events will break experiences.

Reply:
Maybe I am missing something- why would it break the existing experience? All the data are still being send but in a larger chunk (multiple events in one record). Isn't it the implementation details how we are sending data?

Reply:
Oh I see. I missed your previous comment. Make sense.

Reply:
Updated to `simple_aggregation` everywhere.